go.mod

```diff
@@ -10,14 +10,15 @@ require (
 	github.com/docker/docker v24.0.6+incompatible
 	github.com/go-git/go-git/v5 v5.10.0
 	github.com/google/go-cmp v0.6.0
-	github.com/spf13/cobra v1.7.0
+	github.com/spf13/cobra v1.8.0
```

Comment:
Is this necessary?

Reply:
No I don't believe it is. I think I slipped the `go get -u` in on accident--I'll undo it if that's preferred.